web/components/index.sass

```diff
@@ -2,3 +2,4 @@
 @import ./app/App
 @import ./navbar/Navbar
 @import ./input/buttons/index
+@import ./input/TextInput/index
```

Comment:
Why not put this in input/text, instead of its own thing? We’ll have several text input types I imagine (first one that comes to mind is the long form text input) - see how the buttons are set up